src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/CompositionContractMismatchException.cs

```diff
@@ -55,9 +56,7 @@ public CompositionContractMismatchException(string message, Exception innerExcep
             : base(message, innerException)
         {
         }
-
-#if FEATURE_SERIALIZATION
```

Comment:
bring back serialization ctor here.

Reply:
and also in other places which are wrapped with FEATURE_SERIALIZATION as discussed offline.